desktop/release-notes.md

```diff
@@ -24,13 +24,92 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.16.0
+
+{% include release-date.html date="2022-01-12" %}
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Extensions have moved from Beta to GA
+- Quick Search has moved from experimental to GA
+- Extensions are now included in Quick Search
+- Analyzing large images is now up to 4x faster
+
+### Upgrades
+
+- [Compose v2.15.0](https://github.com/docker/compose/releases/tag/v2.15.0)
+- [Containeld v1.6.14](https://github.com/containerd/containerd/releases/tag/v1.6.14)
+- [Docker Engine v20.10.22](https://docs.docker.com/engine/release-notes/#201022)
+- [Docker Scan v0.23.0](https://github.com/docker/scan-cli-plugin/releases/tag/v0.23.0)
+- [Go 1.19.4](https://github.com/golang/go/releases/tag/go1.19.4)
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Fixed `docker build --quiet` not outputting the image identifier with the `containerd` integration.
+- Fixed image inspect not showing image labels with the `containerd` integration.
+- Increased the contrast between running and stopped container icons to make it easier for colourblind people to scan the containers list.
+- Fixed a bug where the user is prompted for new HTTP proxy credentials repeatedly until Docker Desktop is restarted.
+- Added a diagnostics command `com.docker.diagnose login` to check HTTP proxy configuration.
+- Fixed actions on compose stack not working properly. Fixes [docker/for-mac#6566](https://github.com/docker/for-mac/issues/6566).
+- Fixed the Docker Desktop dashboard trying at startup to get disk usage information and display an error banner before the engine was running.
+- Added an informational banner with instructions to opt-out of experimental feature access next to all experimental features.
+- Support downloading Kubernetes images via an HTTP proxy.
+
+#### For Mac
+
+- Fixed the Docker engine not starting when Enhanced Container Isolation is enabled if the legacy `osxfs` implementation is used for file sharing.
+- Fixed files created on VirtioFS having the executable bit set. Fixes [docker/for-mac#6614](https://github.com/docker/for-win/issues/6614).
+- Added back a way to uninstall Docker Desktop from the command line. Fixes [docker/for-mac#6598](https://github.com/docker/for-mac/issues/6598).
+- Fixed hardcoded `/usr/bin/kill`. Fixes [docker/for-mac#6589](https://github.com/docker/for-mac/issues/6589).
+- Fixed truncation (for example with the `truncate` command) of very large files (> 38GB) shared on VirtioFS with an incorrect size.
+
```

Comment:
```suggestion
- Fixed Docker crash under network load. Fixes [docker/for-mac#6530](https://github.com/docker/for-mac/issues/6530).

```

Reply:
What’s the pr number for this one? Did I miss it somehow?

Reply:
@djs55 do you have a link to an internal ticket or PR?

Reply:
POS-1356 I believe

Reply:
Ah, thanks, none of the linked PR had a release note then.